mapie/regression/time_series_regression.py

```diff
@@ -392,19 +394,25 @@ def update(
             return self.partial_fit(X, y, ensemble=ensemble)
         elif self.method == 'aci':
             return self.adapt_conformal_inference(
-                X, y, ensemble=ensemble, alpha=alpha,
+                X, y, ensemble=ensemble, confidence_level=confidence_level,
                 gamma=gamma, optimize_beta=optimize_beta
             )
         else:
             raise ValueError(
                 f"Invalid method. Allowed values are {self.valid_methods_}."
             )
 
-    def predict(
+    # For this ``predict`` function, we have chosen to ignore errors related to
```

Comment:
Suggestion:

`# Overriding _MapieRegressor .predict method here. Bad practise, but this inheritance is questionable and will probably be reconsidered anyway.
`